aten/src/ATen/native/transformers/cuda/flash_attn/fmha_api.h

```diff
@@ -6,6 +6,7 @@
 
 namespace fmha {
 
+TORCH_API
```

Comment:
Why do we expose this symbol now? Was this a problem with the version before?

Reply:
There were build errors that show linking issues for the cpp tests. Adding this should make the symbols visible.